nni/compression/pytorch/compressor.py

```diff
@@ -7,7 +7,7 @@
 from . import default_layers
 
 _logger = logging.getLogger(__name__)
-
+MAX_EPOCHS = 9999
```

Comment:
`MAX_EPOCHS` is defined in `nni/algorithms/compression/pytorch/pruning/constants.py`

Reply:
This constant is useless. Remove it.